dali/base/dafdesc.cpp

```diff
@@ -223,6 +225,9 @@ void ClusterPartDiskMapSpec::fromProp(IPropertyTree *tree)
     }
     replicateOffset = getPropDef(tree,"@replicateOffset",1);
     defaultCopies = getPropDef(tree,"@redundancy",defrep)+1;
+#ifdef _CONTAINERIZED
+    defaultCopies = 1;
```

Comment:
The #ifdefs could do with a comment indicating why this is being done.

Reply:
@ghalliday - have added some comments